package.json

```
@@ -35,6 +35,9 @@
 		"type-check": "tsc --noEmit",
 		"prepack": "pnpm build && clean-pkg-json"
 	},
+	"engines": {
+		"node": ">=14.17"
```

```suggestion
		"node": ">=14.18"
```
https://unpkg.com/browse/rollup@3.28.0/package.json